modules/nw-sriov-configuring-operator.adoc

```diff
@@ -6,21 +6,35 @@
 [id="nw-sriov-configuring-operator_{context}"]
 = Configuring the SR-IOV Network Operator
 
-[IMPORTANT]
+. Create a `SriovOperatorConfig` custom resource (CR) to deploy all the SR-IOV Operator related components.
```

Comment:
Unnumbered bullet when there is only one step. Also a colon at the end. Also is "related" critical, or could that be removed?

```suggestion
* Create a `SriovOperatorConfig` custom resource (CR) to deploy all the SR-IOV Operator components:
```